ateam-common-packets/include/stspin.h

```diff
@@ -0,0 +1,141 @@
+/**
+ * @file motor.h
+ * @author Will Stuckey
+ * @brief 
+ * @version 0.1
+ * @date 2022-06-27
+ * 
+ * @copyright Copyright (c) 2022
+ * 
+ */
+
```

Comment:
Are we going to keep this block up to date? If not, we can probably omit it. (Not a great look when it's already out of date on the first commit ☺️ )

Reply:
I've just added some docs here. Removed some fields. Might clean these out of firmware, but wanna keep this here in case we ever doxygen these definitions. I need to document the structs themselves as well

Reply:
open to differing opinions on this or nuance for fields

Reply:
I'm fine with keeping the docs as long as we make a point of maintaining them.